datenguide_python/output_transformer.py

```diff
@@ -8,42 +8,32 @@
 class QueryOutputTransformer:
     """ IN PROGRESS only region query to do: DOKU """
 
-    def __init__(self):
-        pass
+    def __init__(self, query_response_json: Dict[str, Any]):
```

Comment:
Looks good and forms a basis for making default_list configurable at a later point. We should also eventually modify it to use the return type of the query executioner, namely `ExecutionResults` as input instead of the json_response directly. `ExecutionResults` is defined as follows
```python
class ExecutionResults(NamedTuple):
    query_results: Json
    meta_data: Json
```
and the response jason can simply be extracted as an attribute with `execution_results_instance.query_results`  the advantage is that the execution results also come with the meta data that might be needed or nice to have for some transformations